resources/integration_test/tests/functions.yaml

```diff
@@ -0,0 +1,5 @@
+tests:
+  - description: stub function test
+    skip_reason: "SQL-987: Create ODBC Result Set function tests"
+    test_definition: [  ]
+    db: integration_test
```

Comment:
FYI: There is a missing new line at the end of the file

Reply:
Added newline 👍